packages/core/src/tracing/instrumentation/database/couchbase.js

```diff
@@ -46,6 +48,10 @@ exports.init = function init() {
 // CRUD operations:
 // https://github.com/couchbase/couchnode/blob/e855b094cd1b0140ffefc40f32a828b9134d181c/src/connection_autogen.cpp#L210
 function instrument(cb) {
+  // RawBinaryTranscoder function is added in v4.4.4,
+  // so inorder to check version, we can rely on this logic
+  instrumentV444 = typeof cb.RawBinaryTranscoder === 'function';
```

Comment:
`instrumentV444` represents the latest Couchbase version, right? I suggest using a variable name like `isLatest` or something similar instead, and the comment you added is clear enough.

Reply:
+1

Reply:
👍